### PR TITLE
Deploy and add travis-ci config

### LINKED ISSUE #37: CI/CD for frontend

### LINKED ISSUE #38: Deploy frontend

### PR DESCRIPTION
**Pull request by leahriffell**
### What was changed?
- Created heroku application: https://github.com/facebook/create-react-app
- Added travis for ci/cd 

### Have Tests Been Added?
- [x] No.
- [ ] Yes, but all tests are not passing.
- [ ] Yes, and all are passing.

### Issues:
* Closes #37
* Closes #38

### Tracking Consistency:
- [x] Added appropriate labels
- [x] My code follows the code style of this project and has removed all unnecessary annotations
- [x] I have added comments on my pull request, particularly in hard-to-understand areas
- [x] looked at PR preview to check spelling, syntax, formatting, and completion

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+language: node_js
+node_js:
+- node
+script:
+- npm test
+- npm run build
```
Comment (leahriffell) [inline .travis.yml:6]: I've looked through a lot FE peeps existing travis configs and they don't seem to have any scripts (no test or build). I thought we should add testing before deployment but is this not best practice with a react app?

**Review by damwhit — APPROVED** [review]
If this works, let's go with it! I'd need to do more research, but heroku might be overkill for an SPA since it's a client-side application.

File: .travis.yml (hunk 1 above)
Comment (damwhit) [reply .travis.yml:6]: This looks reasonable for now! Note that I'm using the yarn instead of npm. So you may want to stick with that.

**Review by damwhit — COMMENTED** [review]
If we want to continue using yarn for package management, you should:

1. remove the package-lock.json file (because yarn.lock accomplishes the same thing)
2. Change the scripts to use yarn instead of npm (you may need to make sure travis has yarn installed too)

**Pull request merged**